personio-personnel-data-api-oa3.yaml

```diff
@@ -1267,6 +1277,142 @@ paths:
                       code: 404
                       message: 'The attendance period was not found.'
 
+  /company/attendances/projects:
+    parameters:
+      - $ref: "#/components/parameters/XPersonioPartnerID"
+      - $ref: "#/components/parameters/XPersonioAppID"
+    get:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Provides a list of all company projects.
+      responses:
+        "200":
+          description: ""
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                  data:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/Project"
+    post:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Creates projects into the company account
+      responses:
+        "200":
+          description: ""
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                  data:
+                    $ref: "#/components/schemas/Project"
+        "400":
+          description: "Invalid data was sent"
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                    example: false
+                  error:
+                    type: object
+                    properties:
+                      name:
+                        type: array
+                        items:
+                          type: string
+                          example: "Please introduce a name to continue"
+
+
+    delete:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Creates projects into the company account
```

Comment:
Sometimes you use double quotes (") sometimes not :P ... Let's keep it consistent for new changes

Reply:
Double quotes removed